Added Battle Board to extensions.json

MissingLinkDev opened the pull request:
Make sure that your submission has the following:

- [ X] A link to the raw github markdown file with a YAML front matter definition containing:
  - [X ] The title of your extension
  - [X ] A description of your extension
  - [X ] The author
  - [X ] A hero image
  - [X ] An icon
  - [X ] Tags (these will help with your extension's discoverability)
  - [ X] A link to your extensions manifest

Please, go through these steps before you submit a PR.

- [X ] You have done your changes in a separate branch.

- [X ] You have a descriptive commit message with a short title (first line).

- [ X] You have only one commit (if not, squash them into one commit).

- [X ] Your pull request MUST target the `main` branch on this repository.

- [X ] Your pull request puts your extensions details as the last entry in the extensions.json file

nthouliss commented:
Hi, 

The extension is great, there's only one thing that needs to be edited for submission. Currently the learn-more section on the page goes to your patreon. The learn more link should go to an accessible information page that can help users with documentation (such as the github repo). Linking to patreon is fine but not so much for the learn more section. 

The link to patreon you have in the extension doesn't go to your patreon link correctly atm but this isn't a submission requirement. It's just to let you know :)

MissingLinkDev commented:
> Hi,
> 
> The extension is great, there's only one thing that needs to be edited for submission. Currently the learn-more section on the page goes to your patreon. The learn more link should go to an accessible information page that can help users with documentation (such as the github repo). Linking to patreon is fine but not so much for the learn more section.
> 
> The link to patreon you have in the extension doesn't go to your patreon link correctly atm but this isn't a submission requirement. It's just to let you know :)

Thanks for that, I updated the learn more to point to github repository readme.  And fixed the typo in my patreon link (thanks for the heads up).  store.md has been updated, let me know if I have to do anything else

MissingLinkDev commented:
Added Changr to the pull request, another simple extension

nthouliss commented:
@MissingLinkDev you'll have to raise another PR for the other extension.

nthouliss reviewed (changes requested):
You'll have to raise another PR for the other extension. It's one PR per extension as they are reviewed separately.

MissingLinkDev commented:
> You'll have to raise another PR for the other extension. It's one PR per extension as they are reviewed separately.

Ok, I wouldnt let me do another fork and pull request until this one was done so hoped I could just do them at once.  Ill submit Changr once this one clears

MissingLinkDev commented:
Sorry for the changes, updated hosting provider and needed to update the store.md url
